### ISSUE TITLE
Test on Julia v1.10 (and possibly 1.9)

### ISSUE THREAD

**Issue by sloede** [trixi-framework/TrixiShallowWater.jl#26]
IMHO the main tests should be run on the most recent stable release, plus a smaller testset possibly on older versions to verify that they are still supported.

**Comment by patrickersing** [comment]
We were just about to update the CI to Julia v1.9 in #22.
Since Trixi.jl is still testing on Julia v1.9, do you think it make sense to wait for Trixi.jl to update the testing or should we move directly to v1.10?

**Comment by sloede** [comment]
No, just move directly to v1.10 unless you find hard issues that we need to solve in Trixi.jl first.

**Comment by patrickersing** [comment]
Running the test on Julia  v1.10 some of the sensitive (mostly well-balanced) tests fail just outside their tolerance, see #29 

![image](https://github.com/trixi-framework/TrixiShallowWater.jl/assets/114223904/19f80002-9193-4259-a632-ab3474120193)

Does anyone know of any changes that could have triggered this difference?

Since the results are still correct, I would suggest that we just loosen the tolerance a bit.

**Comment by ranocha** [comment]
I have seen similar differences in SummationByPartsOperators.jl. I guess that they are just caused by some floating point differences coming from different LLVM versions, but I am not 100% certain. However, I would be fine with following your strategy (which I have done also in SummationByPartsOperators.jl).

**Comment by sloede** [comment]
I think I've seen similar floating point differences in https://github.com/trixi-framework/Trixi.jl/pull/1562, where some tests apparently fail due to small changes in the error norms.

**Comment by patrickersing** [comment]
It makes sense that the problem would be caused by some floating point differences as these tests are very sensitive to this.
Then if nobody has any objections, I will just adjust the tolerances in #29 .
